deepcell/layers/padding.py

```diff
@@ -41,7 +41,8 @@ class ReflectionPadding2D(ZeroPadding2D):
     at the top, bottom, left and right side of an image tensor.
 
     Args:
-        padding: int, or tuple of 2 ints, or tuple of 2 tuples of 2 ints.
+        padding (int, or tuple of 2 ints, or tuple of 2 tuples of 2 ints):
```

Comment:
omg does this actually get parsed?

Reply:
Not perfectly
<img width="716" alt="Screen Shot 2020-03-07 at 10 57 26 AM" src="https://user-images.githubusercontent.com/20373588/76150736-78ebc400-6062-11ea-8ec0-3e610fb1c9f7.png">